dashboard/modules/job/job_head.py

```diff
@@ -40,10 +41,18 @@ async def check(self, *args, **kwargs):
         try:
             if not ray.is_initialized():
                 try:
-                    address = self._redis_address
-                    redis_pw = self._redis_password
-                    logger.info(f"Connecting to ray with address={address}, "
-                                f"redis_pw={redis_pw}")
+                    if not use_gcs_for_bootstrap():
+                        ip, port = self._dashboard_head.redis_address
+                        redis_pw = self._dashboard_head.redis_password
+                        address = f"{ip}:{port}"
+                        logger.info(
+                            f"Connecting to ray with address={address}, "
+                            f"redis_pw={redis_pw}")
+                    else:
+                        address = self._dashboard_head.gcs_address
+                        redis_pw = None
+                        logger.info(
+                            f"Connecting to ray with address={address}")
                     ray.init(
                         address=address,
```

Comment:
This only works after we can use gcs_address for address, right?

Reply:
Yes, and that's why job-related tests are disabled right now.